docs/index.html

```diff
@@ -0,0 +1,93 @@
+            <div>
+                <a href="/get-inspired/index.html"><img src="./sprites/22-removebg-preview.png" alt="Get inspired logo"></a>
+            </div>
+            <div class="no-margin">
+                <a  href="/get-inspired/templates/about.html">About</a>
+                <a href="#top">Program</a>
+                <a href="#">Join</a>
+                <a href="#partner">Sponsor</a>
+                <a href="#">News</a>
+                <button type="button">CC Campaign</button>
+            </div>
+        </nav>
+        <header class="header general-margin">
+            <h3 class="h3 overflow">"Hello developers and code lovers!"</h3>
+            <h1 class="h1 overflow">GET INSPIRED BY GREAT SOFTWARE DEVELOPERS</h1>
+        </header>
+        <p class="general-margin center-border  normal-font">
+            A compilation of resources about programmers and their acoomplishments in their career. If you are 
+            looking for successful people to look up to, search for inspiration, or simple
+            admire a good software career path and work.
+        </p>
+        <div class="general-margin">
+            <h2 class="bold-hash overflow">#software #coding #development</h2>
+            <h4 class="bold-hash-2">@ Lovers Of Software development, Art of coding corner</h4>
+        </div>
+    </section>
+    <!-- Section 2 -->
+    <section class="section-2 overflow">
+        <h2 id="top" class="align-self-center section-title">Topics</h2>
+        <div class="guide-bar"></div>
+        <div class="art-hold">
+            <article class="article general-margin"><img class="article-img" src="sprites/emb.png" alt=""><h3 class="text-overflow topic-style">Embedded systems</h3>
+            </article>
+            <article class="article general-margin"><img class="article-img" src="sprites/Protocol.png" alt=""><h3 class="text-overflow topic-style">Protocols</h3>
+            </article>
+            <article class="article general-margin"><img class="article-img" src="sprites/OS.png" alt=""><h3 class="text-overflow topic-style">Operating Systems</h3>
+            </article>
+            <article class="article general-margin"><img class="article-img" src="sprites/progra.png" alt=""><h3 class="text-overflow topic-style">Programming languages</h3>
+            </article>
+            <article class="article general-margin"><img class="article-img" src="sprites/algo.png" alt=""><h3 class="text-overflow topic-style">Algorithms</h3>
+            </article>
+        </div>
+    </section>
+    <!-- Section 4 -->
+     <section class="section-4">
+        <h2 id="partner">Partners</h2>
+        <div class="guide-bar"></div>
+        <div class="brands">
+          <img src="sprites/part1.svg" alt="">
+          <img src="sprites/part2.svg" alt="">
+          <img src="sprites/part3.svg" alt="">
+          <img src="sprites/part4.svg" alt="">
+          <img src="sprites/part5.svg" alt="">
```

Comment:
- [ ] Please add alt text to all images this is important so when browsers are unable to render a page properly a text is displayed 👍🏿